kong/include/kong/services/negotiation/v1/negotiation.proto

```diff
@@ -12,7 +12,7 @@ service NegotiationService {
   // to get which services does the DP handle.
   //
   // Call direction: DP to CP
-  // +wrpc:rpc-id=4
+  // +wrpc:rpc-id=1
```

Comment:
Could you give a short explain for this id? What is the number's principle?

Reply:
I think it is just to sync up with koko?

Reply:
Yes. This was a typo, and koko already fix it.